packages/cli/src/commands/destroy/scaffold/__tests__/scaffoldNoNest.test.js

```diff
@@ -48,21 +57,20 @@ const templateDirectories = templateDirectoryNames.map((name) => {
   })
 })
 const scaffoldTemplates = {}
-templateDirectories.forEach((directory) => {
-  const files = jest.requireActual('fs').readdirSync(directory)
+const actualFs = await vi.importActual('fs-extra')
```

Comment:
Looks like you're sometimes using `importOriginal` and sometimes `importActual`. Is there a difference? Can we be consistent?

Reply:
`importOriginal` is available from the parameters of `vi.mock` but `importActual` is what you can use globally when you're not within the `vi.mock` but still need to get the original unlocked implementation of a package. In this file we mock `fs-extra` up at the top but lower down outside the mock definition we need unmocked access to it.

Reply:
So if it had been up to me I'd probably name the callback parameter `importActual` too then, to match. But I see they name it `importOriginal` in their docs, so makes sense we do that too 👍